files/inputs/test36.json

```diff
@@ -0,0 +1,38 @@
+{
+"fooList": [
+	[
+	    [ 0.2,0.01,0.02,0.04],
```

Comment:
Are you intentionally wrapping the matrix (array of arrays) in an additional outer array (with only one element, the matrix in question)?

Reply:
Hmm, nope! Looks like a copy error on my part. I'll fix this and test to see if that's the issue